modules/nw-operator-cr.adoc

```diff
@@ -284,6 +284,11 @@ endif::operator[]
  While migrating egress traffic, you can expect some disruption to workloads and service traffic until the Cluster Network Operator (CNO) successfully rolls out the changes.
 ====
 
+
+|`gatewayConfig.ipForwarding`
+|`object`
```

Comment:
Maybe rephrase:
Specify `Restricted` to forward all traffic related to OVN-Kubernetes only

to

Specify `Restricted` to only allow IP forwarding for Kubernetes related traffic.

Reply:
@trozet Fixed.